.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+### Submission Checklist:
+
+* [ ] Have you followed the [PR guidelines](../blob/main/content/docs/contributing/to-docs/submit-a-pr.md) including sign-off?
+* [ ] **Use active voice** - Do descriptive sentences have a clear _subject_ _verb_ action?
+* [ ] **Answer the 'why'?** - Does the documentation answer "why does this exist"?
```

Comment:
The argument here is 

> "how to X" should also describe "WHY you would X", otherwise it's incomplete documentation.

Just writing up "hows and whats" seems potentially problematic towards generating great documentation.

Reply:
O3DE is still required to have a manual, which does indeed describe "hows and whats". This is just a different type of documentation and serves a different purpose from conceptually-focused, non-reference work. The rephrasing in the checklist update of the contribution guide changes this to **answer the question at hand**, which is an appropriate way to think about any section in any manual, regardless of taxonomy.